include/dlaf/eigensolver/backtransformation.h

```diff
@@ -0,0 +1,46 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2021, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <blas.hh>
+#include "dlaf/eigensolver/backtransformation/mc.h"
+#include "dlaf/matrix/matrix.h"
+#include "dlaf/types.h"
+#include "dlaf/util_matrix.h"
+
+namespace dlaf {
+namespace eigensolver {
+
+/// Eigenvalue back-transformation implementation on local memory.
+///
+/// It computes Q C, where Q = HH(1) HH(2) ... HH(m-mb)
+/// (HH(j) is the House-Holder transformation (I - v tau vH)
+/// defined by the j-th element of tau and the HH reflector stored in the j-th column of the matrix V.
+///
+/// @param mat_c contains the (m x n) matrix C (blocksize (mb x nb)), while on exit it contains Q C.
+/// @param mat_v is (m x m) matrix with blocksize (mb x mb), which contains the Householder reflectors.
+/// The j-th HH reflector is v_j = (1, V(mb + 1 : n, j)).
```

Comment:
Please, apply the suggestions correctly (see https://github.com/eth-cscs/DLA-Future/pull/339#discussion_r676446149)
```suggestion
/// The j-th HH reflector is v_j = (1, V(mb + j : n, j)).
```